bofire/data_models/strategies/doe.py

```diff
@@ -31,6 +32,11 @@ class DoEStrategy(Strategy):
 
     verbose: bool = False
 
+    objective: OptimalityCriterionEnum = OptimalityCriterionEnum.D_OPTIMALITY
+
+    transform: TransformEnum = TransformEnum.IDENTITY
```

Comment:
Do we actually need this? 

Would it not be sufficient to just have `transform_range: Union[None, Tuple[float, float]] = (-1, 1)` as we are only supporting identity and min max. In case transform range is None we do identitiy scaling, I would then set `None` as default. 

We should also have a validator that checks that in case of the tuple the second number is larger than the first one, I will introduce a type for this.